plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -619,7 +620,7 @@ private void addSplitPluginDependencies(String thisPlugin, MavenRunner.Config mc
             tmp.delete();
         }
         System.out.println("Analysis: coreDep=" + coreDep + " pluginDeps=" + pluginDeps + " pluginDepsTest=" + pluginDepsTest);
-        if (coreDep != null) {
+        if (coreCoordinates != null) {
```

Comment:
There are a couple of other places where coreDep is used:
- When loggin "Analysis: coreDep...".
- When updating the dependencies, to change the version for maven-plugin.

Should those be changed to coreCoordinates too?